app/experimenter/nimbus-ui/src/components/PageRequestReview/DraftStatusOperations.tsx

```diff
@@ -100,15 +102,23 @@ export const DraftStatusOperations = ({
     /* istanbul ignore next until EXP-1055 & EXP-1062 done */
     case DraftStatusOperationsState.LaunchApproveOrReject:
       return (
-        <FormApproveOrRejectLaunch
-          {...{
-            isLoading,
-            launchRequestedByUsername,
-            onApprove: handleLaunchApprovalClicked,
-            onReject: () =>
-              setDraftUIState(DraftStatusOperationsState.LaunchRejection),
-          }}
-        />
+        <>
+          {rsRequestTimedOut && (
```

Comment:
@lmorchard I stuck this in here since we already know here the user has a role of reviewer and we need to render `FormApproveOrRejectLaunch` in both cases. I could create `DraftStatusOperationsState.RSRequestTimedOut` to return this `p` + `FormApproveOrRejectLaunch` if we don't want to intermingle these.

Reply:
Could also move this into `FormApproveOrRejectLaunch` itself and make `rsRequestTimedOut` a prop that that component. Would be nice to keep these case statements as simple and 1-component as possible